apps/dashboard/app/views/widgets/_sessions.html.erb

```diff
@@ -6,7 +6,9 @@
 -%>
 
 <%- unless session_selection.empty? %>
-  <%= javascript_include_tag 'sessions_poller', nonce: true %>
```

Comment:
Do we not need `sessions_poller` at all?

Reply:
I guess we don't as it was the sessions widget the one that was using it.
Will add the `session_poller` code into `batch_connect_sessions.js`

Reply:
This is completed now.

cc: @hmdc/siddev